quadriga/src/main/resources/config.properties

```diff
@@ -6,10 +6,14 @@ db.password=${db.password}
 neo4j.url=${neo4j.url}
 neo4j.database=${neo4j.database}
 
-citesphere_client_id=${citesphere.client.id}
-citesphere_client_secret=${citesphere.client.secret}
-citesphere_base_url=${citesphere.base.url}
+conceptpower_base_url=${conceptpower.base.url}
+conceptpower_id_url=/rest/Concept?id={concept_uri}
+
+citesphere_client_id=OAUTHCLIENT32
+citesphere_client_secret=03c2b326-88ba-4a38-866a-fc6f35ed0423
```

Comment:
these should not be in here (secret information!) and also should be defined in the pom.

Reply:
I regenerated the secret since this is compromised now.